### DISCUSSION TITLE
Tempo transition issues

### DISCUSSION THREAD

**Discussion by sandrojologua** [CoderLine/alphaTab#1268] [General]
There are issues regarding changing tempo progressively. In Guitar Pro 8 automation editor we have constant or progressive tempo transitions. However, this info isn't available in data model alphaTab provides. There is tempoAutomation.isLinear which turns out to be always True.

Thanks.

**Comment by Danielku15** [comment]
alphaTab doesn't provide always full feature parity with Guitar Pro or other formats but rather offers an own data model which can be filled from these formats. The model doesn't contain "everything" from the source formats but rather the parts which are also needed by alphaTab. 

Therefore I wouldn't call it an "issue in alphaTab" but rather "missing functionality for your needs". 

To request new functionalities it would be best to open a dedicated post in the [Ideas](https://github.com/CoderLine/alphaTab/discussions/categories/ideas) section and describe your needs. It is always a good idea to give some background information about what you wanna achieve and build in your integration and not only talk about the solution you think should be added to alphaTab. 

e.g. even if we would fill the `isLinear` property based on some rules it might not be what you want if alphaTab itself doesn't respect the property internally when making tempo transitions.
